plugins/generator-1.20.6/neoforge-1.20.6/templates/dispensebehaviour.java.ftl

```diff
@@ -36,7 +36,7 @@
 package ${package}.item.extension;
 
 <#compress>
-@Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.MOD) public class ${name}ItemExtension {
+@EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.MOD) public class ${name}ItemExtension {

```

Comment:
I have a strong feeling this was not tested at all. This code won't even compile

Reply:
Resolved